schedule.txt

```diff
@@ -0,0 +1,3 @@
+30 54 23 4 14 4 52  obc.time_management.set_time(1649980800)

```

Comment:
I don't think this file should be in the index

Reply:
i don't think it hurts to have it here, just as a reference for the structure of the schedule

Reply:
Files not related to the program shouldn't be checked in